<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widgets::MenuBar</class>
 <widget class="QMenuBar" name="Widgets::MenuBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuBar</string>
  </property>
  <widget class="QMenu" name="menuFile">
   <property name="title">
    <string>File</string>
   </property>
   <addaction name="actionOpen"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionPreferences"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QMenu" name="menuHelp">
   <property name="title">
    <string>Help</string>
   </property>
   <addaction name="actionReport_an_issue"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close file</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReport_an_issue">
   <property name="text">
    <string>Report an issue</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <addaction name="menuFile"/>
  <addaction name="menuHelp"/>
 </widget>
 <resources/>
 <connections/>
</ui>
